projects.md

```diff
@@ -49,15 +49,15 @@ projects:
         link: https://github.com/GBKS/bitcoin-wallet-ui-kit
 
   - name: Bitcoin Design Sprints
-    description: Improving lightning network wallets through applying the design guide, design thinking and problem solving.
+    description: This project from 2022 was about improving lightning network wallets through applying the design guide, design thinking and problem solving.
```

Comment:
just emphasis to point to the archive?

Reply:
I don't understand the question. My rationale here was that it sounds like it's an ongoing project, when it's really gone inactive.

Reply:
Generally, we need to revise that page with more up-to-date projects. Something on my to-do list.

Reply:
I would ignore trying to update this as out of scope for this PR.
2022 wasn't that long ago and who knows if it will forever stay inactive.